gnomad/utils/filtering.py

```diff
@@ -256,3 +256,66 @@ def subset_samples_and_variants(
         f"out of {full_count} samples in MT"
     )
     return mt
+
+
+def filter_to_clinvar_pathogenic(
+    mt: Union[hl.MatrixTable, hl.Table] = None,
+    clnrevstat: str = "CLNREVSTAT",
+    clnsig: str = "CLNSIG",
+    clnsigconf: str = "CLNSIGCONF",
+    remove_no_assertion: bool = True,
+    remove_conflicting: bool = True,
+) -> Union[hl.MatrixTable, hl.Table]:
+    """
+    Returns a MatrixTable or Table that filters the clinvar data to pathogenic and likely pathogenic variants.
+
+    :param: mt: Input dataset that contains clinvar data, could either be a MatrixTable or Table. Defaults to gnomad's grch38 clinvar version.
+    :param clnrevstat: The field string for the expression that contains the review status of the clinical significance of clinvar variants.
+    :param clnsig: The field string for the expression that contains the clinical signifcance of the clinvar variant.
+    :param clnsigconf: The field string for the expression that contains if there is conflicting clinical significance for the variant. Undefined if there is no conflicting signficance.
```

Comment:
The field string for the expression that contains the conflicting clinical significance values for the variant. For variants with no conflicting significance, this field should be undefined.